projects/local_synth_brca/main.py

```diff
@@ -167,8 +196,13 @@ def es2doc(config, sample_size=100):
 
     print(f"Success: {successful_loads}")
     print(f"Failed: {failed_loads}")
-    print(f"Doccano Project ID: {project.id}")
-    return project.id
+    
+    temp1 = {
```

Comment:
Sorry last comment then we can merge. Could we avoid meaningless names please, anywhere.

```suggestion
    doccano_proj_details = {
```